README.md

```diff
@@ -117,7 +117,7 @@ npm install -g homebridge-apple-tv-remote
 
 **name**: The name of the Play Pause button. 
 
-**bundleIdentifier**: The technical name of an app on the AppleTV. This value is reported in the debug information of the plugin. examples: "com.netflix.Netflix" or "com.google.ios.youtube"
+**bundleIdentifier**: The technical name of an app on the Apple TV. This value is reported in the debug information of the plugin. Known bundleIdentifier: "com.netflix.Netflix", "com.google.ios.youtube" and the default apps of the Apple TV: https://support.apple.com/en-us/guide/mdm/mdmc90dce69e/web Please be aware this value is case sensitive. 
```

Comment:
How can the debug information can be enabled so that users can find the bundleIdentifier ?

Reply:
@RaymondMouthaan

You'll get the output when you start homebridge in debug mode. 

You can do this by enabling Debug mode in the interface:
![image](https://user-images.githubusercontent.com/10795733/94405193-7b357d00-0170-11eb-9013-55a120a2a099.png)

Or by running homebridge with -D 
![image](https://user-images.githubusercontent.com/10795733/94405400-c8195380-0170-11eb-94ae-72f9988da089.png)

When doing so the output looks like this: 
![image](https://user-images.githubusercontent.com/10795733/94405678-31996200-0171-11eb-8713-16e84bceb8c2.png)

What we also could do is adding it to the default output of the plugin. I think that is way more obvious for the end user, however does not align with the current output of the plugin. @lukasroegner what do you think?

Reply:
Below a list of bundleIdentifiers I've got so far, this might be handy for users to give them a start :)

```json
"appPlayPauseSwitches": [
{
   "name": "Netflix",
   "bundleIdentifier": "com.netflix.Netflix"
},
{
   "name": "Youtube",
   "bundleIdentifier": "com.google.ios.youtube"
},
{
   "name": "Apple TV",
   "bundleIdentifier": "com.apple.TVWatchList"
},
{
   "name": "Prime",
   "bundleIdentifier": "com.amazon.aiv.AIVApp"
},
{
   "name": "Disney+",
   "bundleIdentifier": "com.disney.disneyplus"
},
{
   "name": "Apple Music",
   "bundleIdentifier": "com.apple.TVMusic"
},
{
   "name": "Apple Airplay",
   "bundleIdentifier": "com.apple.TVAirPlay"
},
{
   "name": "Spotify",
   "bundleIdentifier": "com.spotify.client"
}
]
```

note: if tested these with `homebridge-apple-tv-remote v0.11.0` and they seem to work nicely 💯

Reply:
Thank you for your input! I added it to the list of known bundleIdentifiers